client-v2/src/app/components/atoms/icons/icon/icon.component.ts

```diff
@@ -1,10 +1,15 @@
 import { Component, Input } from '@angular/core'
+import { iconClasses, IconKey, isIconKey } from './icons'
 
 @Component({
     selector: 'app-icon',
-    template: '<i [class]="iconClass"></i>',
+    template: '<i [class]="icon"></i>',
     styleUrls: [],
 })
 export class IconComponent {
-    @Input() iconClass!: string
+    @Input() iconClass!: IconKey | string
```

Comment:
This actually makes more sense the other way around i.e. using `icon` as the input and `iconClass` as the resolved classList